ca_description/urdf/create_2.xacro

```diff
@@ -2,6 +2,8 @@
 <robot name="create_2" xmlns:xacro="http://ros.org/wiki/xacro">
   <xacro:include filename="$(find ca_description)/urdf/create_base.xacro"/>
   <xacro:include filename="$(find ca_description)/urdf/roomblock.xacro"/>
+  <xacro:arg name="robot_number" default="1"/>
```

Comment:
I found [this](https://answers.ros.org/question/282902/pass-parameters-to-xacro-from-launch-file-or-otherwise/) question in stackflow and they tested in the coded, I change the default values to other numbers to check if it was working, and it was.

Reply:
I didn't see this approach in the official documentation but it looks cool.